wallet-new/server/Cardano/Wallet/API/V1/LegacyHandlers/Transactions.hs

```diff
@@ -2,44 +2,48 @@ module Cardano.Wallet.API.V1.LegacyHandlers.Transactions where
 
 import           Universum
 
-import           Cardano.Wallet.API.Request
```

Comment:
I reorganized the imports to have `Prelude -> Libraries -> IOHK packages -> Internal modules` ordering